### DISCUSSION TITLE
Please add Stat Styles / Text Mode option to Bar Gauge visualiztion

### DISCUSSION THREAD

**Discussion by uselessjargon** [grafana/grafana#49108] [Dashboards, Panels & Visualizations]
In a bar gauge with a prom query like sum(aobiz_cust_node_gauge2{custtag=~"cnp"}) by (ostype) the bar gauge displays all 'ostype's in the proper format (ostype name, a pretty bar and a value) when there is more than one 'ostype' to display (customer apim and cnicc in the example screenshot below). However, when there is only a single 'ostype' returned, the display name for the 'ostype' does not appear (selection of customer cnp in screenshot below). The Stat visualization has a Stat Styles / Tex Mode option to override this latter behavior that I would like to have incorporated into the bar gauge visualization.

If you look at the animated gif below you can see how selecting customer cnp produces an ineffective bar gauge. 

![bar-gauge-single-issue](https://user-images.githubusercontent.com/15000796/168878074-c3a6d490-82c9-470c-a84d-ee8739966bdb.gif)

What the Stat visualization has to override the behavior
![image](https://user-images.githubusercontent.com/15000796/168879466-68e1f81c-6993-4f67-b6c9-e4ad285678e4.png)

More detail about the feature can be found in this closed issue.
https://github.com/grafana/grafana/issues/48983#issuecomment-1127434133

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
